core/widgets/SubmitButtonWidget.php

```diff
@@ -40,6 +41,11 @@ class SubmitButtonWidget extends Widget
      */
     public $options = [];
 
+    /**
+     * @var activeForm Define activeForm context to use widget in context and only disable button when validation succeeded
+     */
+    public $activeForm;
```

Comment:
@martinpetrasch please create a setter getter method which can directly verify the object instance:

setActiveFrom(ActiveFrom $activeForm)

(don't forget the @since tags)